source/op/tabulate_multi_device.cc

```diff
@@ -38,6 +39,7 @@ REGISTER_OP("TabulateFusionSeA")
     .Input("table_info: T")
     .Input("em_x: T")
     .Input("em: T")
+    .Input("two_embed: T")
```

Comment:
This is a breaking change. Old models will fail to run.

Please avoid revise old OPs. Instead, add a new OP (maybe called xxV2) that share the same kernel with the old one.

Reply:
Why old models will fail to run? Passing a null pointer to this op will let it fallback to the previous op behaviour.

Reply:
That's how TensorFlow graphs work. When loading the graph from a file, the code to define the graph will never be visited again.